contracts/HasIntents.sol

```diff
@@ -0,0 +1,32 @@
+/* solhint-disable-next-line compiler-fixed */
+pragma solidity ^0.4.23;
+
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+// Common: HasIntents
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+
+contract HasIntents {
```

Comment:
Is this contract name intentional or it's HashIntents?

Reply:
I have the same question. BTW why not just Intents ?

Reply:
Initially, I had given it exactly that name, but on the argument that I made above regarding its purpose based on @benjaminbollen advice I changed it to `HasIntents`.

Reply:
I take the blame for that, but I neither like `MyContract is Intents`, or `MyContract is HasIntents`; we're misusing the inheritance model here a bit.  I suggested `HasIntents` in line with `HasWorkers` where we also produce bad english to express that it has an object, rather than is a thing...
improvement suggestions for both are very much welcome

Reply:
@benjaminbollen could we use `MapsIntents` or `MapIntents`?